velocity.js

```diff
@@ -20,10 +35,18 @@ app.use(session({
     }
 }));
 
+app.use(function(req, res, next) {
+    res.locals.__ = res.__ = function() {
+        return i18n.__.apply(req, arguments);
+    };
+
+    next();
+});
+
 app.listen(config.port, function () {
     console.log(`app is listening on port ${config.port}`);
 });
 
 app.get('/', function (req, res) {
-    return res.status(200).send('hello world');
+    return res.status(200).render('login');
```

Comment:
move login as a constant at the top

Reply:
is that going to be a new convention?

Reply:
yeah, I ran into a lot of issue with quizzard regarding this